documentatie/flowcharts/data-entry-forms.md

```diff
@@ -0,0 +1,219 @@
+# Data Entry Forms
+
+This page describes the navigation and rendering logic of the data entry forms.
+
+An important thing to keep in mind when reading these diagrams is that a user can only move on to the next form after resolving all errors (if any) and accepting all warnings (if any). So it should not be possible that any form previous to the current one has any errors or unaccepted warnings.
+
+
+## Questions
+- Where does "cache input" happen in these flows? After each keystroke?
+
+
+## Render navigation menu
+
+Render happens based on last received API response.
+
+```mermaid
+flowchart TD
+    %% elements
+    flow-start([start])
+
+    disabled-styling([disabled styling])
```

Comment:
het duurde even voordat ik dit las als 'styling for disabled state' ipv 'styling is disabled'. Misschien iets preciezer bewoorden, zie suggestie. Dit geldt ook voor de andere states.
```suggestion
    disabled-styling([styling for disabled state])
```

Reply:
Ga er "styling: disabled" etc. van maken. Is korter en hopelijk ook voldoende duidelijker dan wat het was.